### PR TITLE
Introduce aliases of empty/exist

### PR DESCRIPTION
**Pull request by vesln**
See https://github.com/airbnb/enzyme/issues/12#issuecomment-169374132

/cc @ljharb @andreasklinger @ayrton

### CODE REVIEW

**Comment by ljharb** [comment]
awesome, LGTM :-)

**Comment by andreasklinger** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -15,8 +15,8 @@
   1. [contain(node)](#containnode)
   1. [descendants(selector)](#descendantsselector)
   1. [disabled()](#disabled)
-  1. [empty](#empty)
-  1. [exist](#exist)
+  1. [blank()](#blank)
+  1. [present()](#present())
```
Comment (ayrton) [inline README.md]: I think this should be `[present()](#present)`

**Comment by vesln** [comment]
@ljharb just granted you access to the repo. lmk if there are other people i should add

File: README.md (hunk 1 above)
Comment (vesln) [inline README.md]: :+1:

**Comment by ljharb** [comment]
@vesln thanks! if you wouldn't mind, please add @lelandrichardson as well, since he's the primary enzyme maintainer.

**Comment by vesln** [comment]
@lelandrichardson just added u as well, cheers!

**Comment by andreasklinger** [comment]
@ljharb @lelandrichardson welcome folks :)

**Pull request merged**